File: 2st/Riyeon/K번째수.java
import java.util.*;
class Solution {
    public int[] solution(int[] array, int[][] commands) {
        int[] answer = new int[commands.length];
        int k=0;
        for(int[] cm : commands){
            int[] arr = Arrays.copyOfRange(array, cm[0]-1, cm[1]);
            Arrays.sort(arr);
            answer[k++] = arr[cm[2]-1];
        }
        return answer;
    }
}

File: 4st/Seohyun/K번째수.js
function solution(array, commands) {
  let answer = [];

  for (let i = 0; i < commands.length; i++) {
    let ii = commands[i][0];
    let jj = commands[i][1];
    let kk = commands[i][2];

    let arr = array.slice(ii - 1, jj);

    arr.sort((a, b) => a - b);

    answer.push(arr[kk - 1]);
  }

  return answer;
}
